releasenotes/releasenotes.rst

```diff
@@ -24,6 +24,25 @@ Public Endpoint Announcement 8 July 2021
 
 The default FHIR version of the `public Firely Server endpoint <https://server.fire.ly/>`_ is now R4.
 
+.. _vonk_releasenotes_421:
```

Comment:
I would add a note saying that we needed to fix due to an error in 4.2.0

Reply:
Also add a comment about what was wrong in 4.2.0

Reply:
Thanks for comments. I have updated the release notes. Could you check it again? Please feel free to suggest otherwise. :-)

Reply:
I'll mark this as resolved since Lennart is helping already.